### DISCUSSION TITLE
iOS login replication

### DISCUSSION THREAD

**Discussion by notmarek** [notmarek/BeFake#71] [General]
I have successfully unpinned the certificates from the iOS bereal app, unfortunately it still uses the same login api [(this)](https://developers.google.com/resources/api-libraries/documentation/identitytoolkit/v3/csharp/latest/classGoogle_1_1Apis_1_1IdentityToolkit_1_1v3_1_1Data_1_1IdentitytoolkitRelyingpartySendVerificationCodeRequest.html) which according to the documentation transfers the iosReceipt, and iosSecret trought the Apple APNS (notification system) which as far as i know theres no way to emulate and force google to send it to us.

**Comment by valerierx** [comment]
Yeah, reverse engineering that would be far more troublesome than anything. Lets hope BeReal keeps reCAPTCHA and vonage running.

**Comment by notmarek** [comment]
It would be almost impossible because it uses the iOS notification system.
